.rubocop.yml

```diff
@@ -0,0 +1,21 @@
+inherit_gem:
+  betterlint:
+    - config/default.yml
+
+inherit_mode:
+  merge:
+    - Exclude
+
+AllCops:
+  TargetRubyVersion: 2.6
```

Comment:
We could probably do something more recent, but that would require updating `spec.required_ruby_version` in the gemspec as well.

Reply:
how recent you think we should go?

Reply:
That will likely produce some additional linter flags and autocorrects.